### ISSUE TITLE
Yeelight is always unreachable in Hue App but works nevertheless

### ISSUE THREAD

**Issue by betonishard** [diyhue/diyHue#215]
**Describe the bug**
Yeelight bulb show always unreachable but it work perfectly fine
**To Reproduce**
1. Go to Hue app
2. See that Yeelight bulb is "unreachable"
3. Change color, you will notice that it works immediately
4. Leave app, get back in the app and it's black (before it did not remember the correct color and returned always RED

**Expected behavior**
Don't show unreachable if it 's not unreachable, and show the correct color (the last used color)
**Logs**
Below a picture of the log.

`2019-10-05 18:29:09,965 - root - INFO - YEELIGHT name is: BankADDREESS10.1.0.165
2019-10-05 18:29:09,971 - root - WARNING - Bank is unreachable: can only concatenate str (not "list") to str
10.1.0.162 - - [05/Oct/2019 18:29:10] "GET /api/100a5f3ae2bf11e99d8ae2419c18002e/lights HTTP/1.1" 200 -
10.1.0.162 - - [05/Oct/2019 18:29:10] "GET /api/100a5f3ae2bf11e99d8ae2419c18002e/sensors HTTP/1.1" 200 -`

**Additional context**

**Comment by alexyao2015** [comment]
The color issue relates to #181

**Comment by alexyao2015** [comment]
I think you need to clear your config. You have something in there that isn't quite right. Yeelight works fine. Color is a problem for all lights currently.

**Comment by betonishard** [comment]
@alexyao2015 thank you for your response. WOuld the correct approach be to delete the current config.json, create a new one with the basic info provided in the repo? Would you also delete the app along with its data as well?

**Comment by alexyao2015** [comment]
Yes that is exactly what I would do. Delete the entire config.json and cert.pem and clear application data.

**Comment by betonishard** [comment]
@alexyao2015 I will do a full rm of the directory and easy install it again. Will report back on this thread..

**Comment by alexyao2015** [comment]
Any progress on this?

**Comment by betonishard** [comment]
Unfortunately no, charger port of my server broke down last week.. Hopefully this weekend fixed.

**Comment by betonishard** [comment]
@alexyao2015 I just made all the updates and can confirm that unreachable is n ot there anymore (nevertheless wrong startup color on yeelight is but that's another topic),.

**Comment by alexyao2015** [comment]
That is #181
